### ISSUE TITLE
Avast virus detection

### ISSUE THREAD

**Issue by Double44** [gurnec/HashCheck#25]
Likely a false positive. I'm pretty sure I installed the app though, but it seems to be missing. It doesn't exist in the Control Panel, which seems to suggest that it was either auto-uninstalled or quarantined altogether.

https://i.imgur.com/VSGUvdp.png
https://i.imgur.com/HRASyQE.png

**Comment by ghost** [comment]
Yes, it is most likely a false positive as VirusTotal reports nothing.
